### PR TITLE
Fix version incompatibility

### LINKED ISSUE #17: PyVISA-py 0.5.0 requires pyvisa>=1.11.0
A `conda update --all` broke my environment. After the release of pyvisa-py 0.5.0 the dependency requirements were fixed, see pyvisa/pyvisa-py@112c577131c516ac066c19b86994ff159cfe7451. This package (version 0.5.0) should require pyvisa>=1.11.0. Thanks!

### PR DESCRIPTION
**Pull request by davidfokkema**
pyvisa-py requires pyvisa >= 1.11. Fixes #17. Without this change, a `conda install pyvisa-py` results in a broken installation.

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by davidfokkema** [comment]
@conda-forge-admin, please rerender

**Comment by mtsolmn** [comment]
Hi @davidfokkema, I am aware of the issue and the need to update the pyvisa-py feedstock. Unfortunately, in order to do so the pyvisa feedstock needs to be updated first. With a slow response from the maintainers, I attempted to update it myself https://github.com/conda-forge/pyvisa-feedstock/pull/24, however there is an unresolved issue with the windows build process. I've tried to get the conda-forge admins attention to help with this to no avail.

As soon as pyvisa is updated I will push an update to pyvisa-py. Thanks.

**Comment by davidfokkema** [comment]
I was afraid the checks would fail because the dependencies are currently unsolvable. However, if you were to push this build, conda should always install pyvisa 1.10.1 and pyvisa-py 0.4.1, which is a working combination. I don't know if this is possible. By waiting on pyvisa to be updated we cannot do `conda install pyvisa-py`. My students are doing that en masse and I have to keep telling them to fix it by `conda install pyvisa-py=0.4.1`. This should not happen.

So: I know you're aware of the problem, and I know that pyvisa should be updated. But can this requirement be pushed in the mean time so that `conda install pyvisa-py` will result in a working installation?

**Comment by MatthieuDartiailh** [comment]
https://github.com/conda-forge/pyvisa-feedstock/pull/26 has now been merged (sorry the issue we hit was quite stupid but hard to parse). I will try to re-trigger a build tomorrow and hopefully merge after.

**Comment by MatthieuDartiailh** [comment]
@conda-forge-admin , please rerender

**Comment by mtsolmn** [comment]
@conda-forge-admin , please rerender
